src/main/java/com/rapid7/client/dcerpc/mssamr/SAMPRDomainLogOffInfo.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2017, Rapid7, Inc.
+ *
+ * License: BSD-3-clause
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *   Redistributions of source code must retain the above copyright notice,
+ * this list of conditions and the following disclaimer.
+ *
+ *  Redistributions in binary form must reproduce the above copyright
+ * notice, this list of conditions and the following disclaimer in the
+ * documentation and/or other materials provided with the distribution.
+ *
+ *  Neither the name of the copyright holder nor the names of its contributors
+ * may be used to endorse or promote products derived from this software
+ * without specific prior written permission.
+ *
+ *
+ */
+
+package com.rapid7.client.dcerpc.mssamr;
+
+import java.io.IOException;
+import java.util.Objects;
+import com.rapid7.client.dcerpc.io.PacketInput;
+import com.rapid7.client.dcerpc.io.ndr.Unmarshallable;
+
+/* typedef struct _DOMAIN_LOGOFF_INFORMATION {
+ *   OLD_LARGE_INTEGER ForceLogoff;
+ * } DOMAIN_LOGOFF_INFORMATION,
+ *  *PDOMAIN_LOGOFF_INFORMATION;
+ */
+public class SAMPRDomainLogOffInfo implements Unmarshallable {
+
+    private int forceLogoff;
+
+    public int getForceLogoff() {
+        return forceLogoff;
+    }
+
+    public void setForceLogoff(int forceLogoff) {
+        this.forceLogoff = forceLogoff;
+    }
+
+    @Override
+    public void unmarshalPreamble(PacketInput in) throws IOException {
+    }
+
+    @Override
+    public void unmarshalEntity(PacketInput in) throws IOException {
```

Comment:
You need to align the structure. This structure's alignment is 8, as the alignment of `OLD_LARGE_INTEGER` is 4 (2x 4 byte fields).
```
@Override
public void unmarshalEntity(PacketInput in) throws IOException {
    // Structure Alignment: 4
    in.align(Alignment.FOUR);
```

Reply:
@tosmun-r7 , done! it worked

Reply:
yep it was updated ...